src/main/resources/liquibase/DDP-4702_AT-form.xml

```diff
@@ -0,0 +1,481 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+        <changeSet author="simone" id="atForm">
+            <insert tableName="field_settings">
+                <column name="ddp_instance_id"
+                        valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE instance_name='atcp')"></column>
+                <column name="field_type" value="TAB"/>
+                <column name="column_name" value="AT_GROUP_ELIGIBILITY"/>
+                <column name="column_display" value="Eligibility"/>
+                <column name="display_type" value="TAB"/>
+                <column name="order_number" value="2"/>
+            </insert>
+            <insert tableName="field_settings">
+                <column name="ddp_instance_id"
+                        valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE instance_name='atcp')"></column>
+                <column name="field_type" value="AT_GROUP_ELIGIBILITY"/>
+                <column name="column_name" value="ELIGIBILITY"/>
+                <column name="column_display" value="Eligibility"/>
+                <column name="display_type" value="RADIO"/>
+                <column name="possible_values" value="[{&#34;value&#34;:&#34;2&#34;,&#34;name&#34;:&#34;Pending&#34;},{&#34;value&#34;:&#34;1&#34;,&#34;name&#34;:&#34;Eligible&#34;},{&#34;value&#34;:&#34;0&#34;,&#34;name&#34;:&#34;Not Eligible&#34;}]"/>
```

Comment:
In the future I would use xml way of escaping quotes which is `&quot;` instead of `&#34;` which would be more easier to find if some typo happened